NIPostsReload/Presentation/Scenes/NIPostFeedModule/Presenter/NIPostFeedPresenter.swift

```diff
@@ -50,6 +53,19 @@ final class DefaultNIPostFeedPresenter: NIPostFeedPresenter {
         postViewState.posts.count
     }
     
+    func getSelectedFeedType() -> PostFeedType {
+        selectedFeedType
```

Comment:
You can use property getter instead of the additional method

var selectedFeedType: PostFeedType { get }